pallets/did/src/did_details.rs

```diff
@@ -670,37 +670,37 @@ pub trait DeriveDidCallAuthorizationVerificationKeyRelationship {
 /// extrinsic to have a DID origin and perform DID-based authorization upon
 /// their invocation.
 #[derive(Clone, RuntimeDebug, Decode, Encode, PartialEq, TypeInfo)]
-#[scale_info(skip_type_params(T))]
 
-pub struct DidAuthorizedCallOperation<T: Config> {
+pub struct DidAuthorizedCallOperation<DidIdentifier, DidCallable, BlockNumber, AccountId> {
 	/// The DID identifier.
-	pub did: DidIdentifierOf<T>,
+	pub did: DidIdentifier,
 	/// The DID tx counter.
 	pub tx_counter: u64,
 	/// The extrinsic call to authorize with the DID.
-	pub call: DidCallableOf<T>,
+	pub call: DidCallable,
 	/// The block number at which the operation was created.
-	pub block_number: BlockNumberOf<T>,
+	pub block_number: BlockNumber,
 	/// The account which is authorized to submit the did call.
-	pub submitter: AccountIdOf<T>,
+	pub submitter: AccountId,
 }
 
 /// Wrapper around a [DidAuthorizedCallOperation].
 ///
 /// It contains additional information about the type of DID key to used for
 /// authorization.
 #[derive(Clone, RuntimeDebug, PartialEq, TypeInfo)]
-#[scale_info(skip_type_params(T))]
 
-pub struct DidAuthorizedCallOperationWithVerificationRelationship<T: Config> {
+pub struct DidAuthorizedCallOperationWithVerificationRelationship<DidAuthorizedCallOperation> {
```

Comment:
This probably shouldnot be generic tho. It's tightly coupled to the `DidAuthorizedCallOperation` type.